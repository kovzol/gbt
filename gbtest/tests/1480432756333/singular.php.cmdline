#!/bin/sh
cd tests/1480432756333
cat singular.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/Singular > singular.php.out 2>singular.php.err