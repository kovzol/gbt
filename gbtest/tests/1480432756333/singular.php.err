0.00user 0.00system 0:00.00elapsed 100%CPU (0avgtext+0avgdata 9680maxresident)k
0inputs+8outputs (0major+392minor)pagefaults 0swaps
