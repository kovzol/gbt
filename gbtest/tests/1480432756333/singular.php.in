ring r=0,(v5,v6),dp;ideal I=2+5*v6+-1*v6^2+2*v5+-1*v5^2; option(redSB); groebner(I);