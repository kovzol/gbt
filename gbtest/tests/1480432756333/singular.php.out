                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /   version 4.0.3
                                                           0<
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \   Jan 2016
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \  Debian 4.0.3+ds-1
_[1]=v5^2+v6^2-2*v5-5*v6-2
> Auf Wiedersehen.
